src/core/loader/ncch.cpp

```diff
@@ -246,20 +243,12 @@ ResultStatus AppLoader_NCCH::LoadExeFS() {
     if (!file.IsOpen())
         return ResultStatus::Error;
 
-    // Reset read pointer in case this file has been read before.
-    file.Seek(0, SEEK_SET);
+    // Reset read pointer to ncch_offset case this file has been read before.
+    file.Seek(ncch_offset, SEEK_SET);
```

Comment:
in that case I'd expect it to clear all other temp variables because I would not expect that order of commands like this would cause any different behaviour.

Where is `ncch_offset` set / how visible is it?